css/index.html

```diff
@@ -1,5 +1,6 @@
 <html>
   <head>
+    <link rel="stylesheet" href="https://unpkg.com/@nenkan/css@0.8.0/dist/style.css" >
```

Comment:
Could you move the `<link>` tag to before the end of `</head>`, right after the `<title>` line?